docs/security.html

```diff
@@ -1275,10 +1275,8 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
     <h3 class="anchor-heading"><a id="security_authz" class="anchor-link"></a><a href="#security_authz">7.5 Authorization and ACLs</a></h3>
     Kafka ships with a pluggable authorization framework, which is configured with the <code>authorizer.class.name</code> property in the server configuration.
     Configured implementations must extend <code>org.apache.kafka.server.authorizer.Authorizer</code>.
-    Kafka provides default implementations which store ACLs in the cluster metadata (either Zookeeper or the KRaft metadata log).
+    Kafka provides default implementations which store ACLs in the cluster metadata (KRaft metadata log).
```

Comment:
`default implementations` -> `a default implementation`, since there's just one now

Reply:
Thanks. Updated it.